tests/gis_tests/inspectapp/tests.py

```diff
@@ -133,12 +133,12 @@ def test_time_field(self):
         ))
 
         # The ordering of model fields might vary depending on several factors (version of GDAL, etc.)
-        self.assertIn('    f_decimal = models.DecimalField(max_digits=0, decimal_places=0)', model_def)
+        self.assertIn('    f_decimal = models.CharField(max_length=0)', model_def)
```

Comment:
@claudep, do you think this issue could be something similar to ffdf507ec0821f0520e315c2e8a6cf231ea3fd5a but for float/double?